warehouse/integrations/redshift/redshift_test.go

```diff
@@ -93,9 +96,21 @@ func TestIntegrationRedshift(t *testing.T) {
 			credentials, err := testhelper.RedshiftCredentials()
 			require.NoError(t, err)
 
-			db, err := redshift.Connect(credentials)
+			dsn := url.URL{
+				Scheme: "postgres",
+				User:   url.UserPassword(credentials.Username, credentials.Password),
+				Host:   fmt.Sprintf("%s:%s", credentials.Host, credentials.Port),
+				Path:   credentials.DbName,
+			}
+			params := url.Values{}
+			params.Add("sslmode", "require")
+
+			dsn.RawQuery = params.Encode()
+			db, err := sql.Open("postgres", dsn.String())
```

Comment:
dsn := fmt.Sprintf(
		"postgres://%s:%s@%s:%s/%s?sslmode=disable",
		credentials.User, credentials.Password, credentials.Host, credentials.Port, credentials.DBName,
	)
Maybe we can use a simpler version here?